linux-anvil/entrypoint

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash -il
 
+# Create conda user with the same uid as the host, so the container can write
+# to mounted volumes
+# Adapted from https://denibertovic.com/posts/handling-permissions-with-docker-volumes/
+USER_ID=${HOST_USER_ID:-9001}
+useradd --shell /bin/bash -u $USER_ID -o -c "" -m conda
+export HOME=/home/conda
+chown -R conda /opt/conda
+
 # Source everything that needs to be.
 . /opt/docker/bin/entrypoint_source
 
 # Run whatever the user wants.
-exec "$@"
+exec /usr/local/bin/gosu conda "$@"
```

Comment:
There's a couple other environment variables I missed. Not sure if we care or not.

```
MAIL=/var/spool/mail/conda
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/conda/bin
```

Not really worried about `MAIL` (though it's an easy fix). Kind of nervous about `PATH` as it has `/root/bin` in it currently. If we fix it before the activation step, the above should work.

There must be a better way than fixing these environment variable like this, but it isn't obvious to me ATM.